lib/util.js

```diff
@@ -137,7 +187,10 @@ function timestamp() {
 }
 
 let console;
-// Log is just a thin wrapper to console.log that prepends a timestamp
+/**
+ * Log is just a thin wrapper to console.log that prepends a timestamp
+ * @type {(...args: any[]) => void}
+ */
 function log(...args) {
```

Comment:
I think `log` is deprecated since v6.0.0.
I am not sure about this one.

Reply:
You are right https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_util_log_string